lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -603,7 +603,6 @@ public static function get_merged_data( $origin = 'custom' ) {
 		$result = new WP_Theme_JSON_Gutenberg();
 		$result->merge( static::get_core_data() );
 		if ( 'default' === $origin ) {
-			$result->set_spacing_sizes();
```

Comment:
Moved to [`WP_Theme_JSON_Gutenberg->__construct()`](#diff-b03597cc3da199e5aa5a94950e8241135904853e7c3b82d758e42744878afae7R750) so that the generated presets for each origin are merged properly rather than always being set as `default` origin.